angular.json

```diff
@@ -133,6 +133,10 @@
           }
         }
       }
-    }},
-  "defaultProject": "canvasboard"
-}
+    }
+  },
+  "defaultProject": "canvasboard",
+  "cli": {
+    "analytics": "f07f4b3b-52e3-4a35-8e03-3c8547e12d26"
```

Comment:
Why this parameter?

Reply:
I haven't added it manually. It came when I downloaded the angular cli.